ledger-bridge.js

```diff
@@ -84,10 +85,17 @@ export default class LedgerBridge {
                     this.app = new LedgerEth(this.transport)
                 }
             }
-            else {
+            // This is the approach used to check for u2f support in the library that the ledger transport relies on https://github.com/grantila/u2f-api/blob/master/lib/u2f-api.ts#L76
+            else if (typeof window.u2f !== 'undefined') {
                 this.transport = await TransportU2F.create()
                 this.app = new LedgerEth(this.transport)
             }
+            else if ('hid' in navigator) {
```

Comment:
Nit: There is a proposal under consideration in the ESLint config repository that would ban the use of the `in` operator because it traverses the entire prototype chain, which can be unexpected and confusing.

The alternatives I usually suggest are `Object.keys(obj).includes(key)` or `Reflect.prototype.hasOwnProperty.call(object, key)`, but in this case I think an `undefined` check would suffice.

```suggestion
            else if (navigator.hid !== undefined) {
```